.travis.yml

```diff
@@ -1,22 +1,41 @@
-language: python
-python:
-  - "3.6"
-  - "3.7"
-  - "3.8"
-addons:
-  apt:
-    update: true
-env:
-  - PACKAGE_NAME=pytest-confluence-report PACKAGE_VERSION=0.0.2
-before_install:
-  - sudo add-apt-repository -y ppa:duggan/bats
-  - sudo apt-get -y install bats
-install:
-  - pip install pip -U && pip install -r requirements-dev.txt -U
-script:
-  - bats test-package.bats
-  - ./analyse-source-code.sh
-after_success:
-  - coveralls
-notifications:
-  email: false
+build:
+  language: python
+  python:
+    - "3.6"
+    - "3.7"
+    - "3.8"
+  addons:
+    apt:
+      update: true
+  env:
+    - PACKAGE_NAME=pytest-confluence-report PACKAGE_VERSION=0.0.2
+  before_install:
+    - sudo add-apt-repository -y ppa:duggan/bats
+    - sudo apt-get -y install bats
+  install:
+    - pip install pip -U && pip install -r requirements-dev.txt -U
+  script:
+    - bats test-package.bats
+    - ./analyse-source-code.sh
+  after_success:
+    - coveralls
+  notifications:
+    email: false
+
+test:
+  language: python
```

Comment:
I would highly recommend moving all duplicate parts into a single place to be able to reuse it afterward. E.g here we have languages versions and addons the same in both stages, it would be difficult to maintain it.

Reply:
moved to initial part of the yaml